automation/vm/deploy-vm.ps1

```diff
@@ -0,0 +1,70 @@
+[CmdletBinding()]
+param (
+    [Parameter(Mandatory=$True)]
+    [string]
+    $SubscriptionId,
+
+    [Parameter(Mandatory=$True)]
+    [string]
+    $TenantId,
+
+    [Parameter(Mandatory=$False)]
+    [string]
+    $ServicePrincipalId,
+
+    [Parameter(Mandatory=$False)]
+    [SecureString]
+    $ServicePrincipalPassword,
+
+    [Parameter(Mandatory=$True)]
+    [string]
+    $RGName,
+    
+    [Parameter(Mandatory=$True)]
+    [string]
+    $Location,
+
+    [Parameter(Mandatory=$True)]
+    [string]
+    $TempFilePath,
+
+    [Parameter(Mandatory=$True)]
+    [string]
+    $ParamFilePath,
+
+    [Parameter(Mandatory=$True)]
+    [string]
+    $VMName,
+
+    [Parameter(Mandatory=$True)]
+    [string]
+    $vnet,
+
+    [Parameter(Mandatory=$True)]
+    [string]
+    $subnet,
+
+    [Parameter(Mandatory=$True)]
+    [string]
+    $sgname,
+
+    [Parameter(Mandatory=$True)]
+    [string]
+    $ipaddress
+    
+)
+
+#To login to azure from powershell use the following
+Connect-AzAccount 
+#to Connect to Azure using a service principal account
+    
+  #  $Credential = Get-Credential
+  #  Connect-AzAccount -Credential $Credential -Tenant 'xxxx-xxxx-xxxx-xxxx' -ServicePrincipal
```

Comment:
I think unused code comments to the final merge better be removed

Reply:
removed